{"links": {"self": {"href": "data/repositories/labscript_suite/labscript_devices/pullrequests/81/comments/114766153.json"}, "html": {"href": "#!/labscript_suite/labscript_devices/pull-requests/81/_/diff#comment-114766153"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 81, "links": {"self": {"href": "data/repositories/labscript_suite/labscript_devices/pullrequests/81.json"}, "html": {"href": "#!/labscript_suite/labscript_devices/pull-requests/81"}}, "title": "User devices"}, "content": {"raw": "Correction: I'm only getting double import errors because I have put some\nlabscript basic type subclasses in the user_devices top level `__init__.py`\nfile. If I import devices from user_devices before that import all is good.\nIf I import after, the double import denier gets me. Do I have to move\nthose definitions to their own submodule?\n", "markup": "markdown", "html": "<p>Correction: I'm only getting double import errors because I have put some\nlabscript basic type subclasses in the user_devices top level <code>__init__.py</code>\nfile. If I import devices from user_devices before that import all is good.\nIf I import after, the double import denier gets me. Do I have to move\nthose definitions to their own submodule?</p>", "type": "rendered"}, "created_on": "2019-08-28T12:15:18.516756+00:00", "user": {"display_name": "David Meyer", "uuid": "{8df655d6-8661-4957-b4ba-669cef19bf2d}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B8df655d6-8661-4957-b4ba-669cef19bf2d%7D"}, "html": {"href": "https://bitbucket.org/%7B8df655d6-8661-4957-b4ba-669cef19bf2d%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2a4efe81a17e5aeccdbc4ef2ca2cc463d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsDM-0.png"}}, "nickname": "dihm", "type": "user", "account_id": "557058:411f7712-e1fa-438c-b578-0359da2d9f06"}, "updated_on": "2019-08-28T12:15:21.589484+00:00", "type": "pullrequest_comment", "id": 114766153}